README.md

```diff
@@ -5,11 +5,89 @@ Usage for the example project:
 1. Pull the Solidity source files from git: `npx ts-node src/index.ts pull nabla`
 2. Deploy: `npx ts-node src/index.ts deploy nabla --network foucoco`
 
-Alternatively you can use the parameter `local` instead of `foucoco`. This expects that there is a local chain running on port `9944` – this is particularly useful to run together with the [foucoco-standalone](https://github.com/pendulum-chain/foucoco-standalone) node.
+Alternatively you can use the parameter `local` instead of `foucoco`. This expects that there is a local chain running
+on port `9944` – this is particularly useful to run together with
+the [foucoco-standalone](https://github.com/pendulum-chain/foucoco-standalone) node.
 
 # Required
 
 ```
 brew install binaryen
 ```
 
+## Install solang
+
+You can find the installation instructions for solang [here](https://solang.readthedocs.io/en/v0.3.3/installing.html).
+
+### Build from source
+
+If you want to test a specific version, you can build solang from source.
+
+#### Install Rust
+
+Make sure you have at least rust v1.74 or later installed.
+You can install it with rustup using
+
+```shell
+rustup install 1.74
+rustup default <that_version>
+```
+
+#### LLVM
+
+You need to have a custom version of LLVM available.
+To install it, please follow the
+steps [here](https://solang.readthedocs.io/en/v0.3.3/installing.html#step-1-install-the-llvm-libraries).
+
+#### Build solang
+
+Clone the solang repository and build it:
+
+```shell
```

Comment:
I think that the path to the solang binary used [here](https://github.com/pendulum-chain/wasm-deploy/blob/ad085183d1698ce4ac43aa75a7b9126b1663b01c/src/actions/compileContract.ts#L77) should be absolute.

Reply:
But if we make it absolute then every developer definitely needs to touch that path. If we keep it relative we just have to make sure that the directory is put into the conventional place.